src/components/activity/ActivityDirectivesTable.svelte

```diff
@@ -210,6 +274,16 @@
       <ContextMenu.Item size="sm" on:click={scrollTimelineToActivityDirective}>Scroll to Activity</ContextMenu.Item>
       <ContextMenu.Separator />
     {/if}
+  </svelte:fragment>
+
+  <svelte:fragment slot="context-menu-alt">
```

Comment:
Is there a reason to have this `context-menu-alt` slot instead of just moving the bulk shift menu into the `context-menu` slot above?

Reply:
Oh true. My initial comment on not moving the slot in the first place might have misled me into thinking to just add a new slot. I agree with @AaronPlave, if the new context menu items can just be added to the slot above, that would be preferred.

Reply:
per discussion on other PR, we decided to keep this additional slot and rename to 'context-menu-bottom' for clarity